plugins/@grouparoo/hubspot/__tests__/.env.example

```diff
@@ -14,3 +14,6 @@ HUBSPOT_API_KEY = "a31ef454-1835-4b88-8d3a-eefc0b688ded"
 # last_name                   (String)
 # text_field                  (String)
 # number_field                (Number)
+#The Hubspot account should have 2 companies with the following domains:
```

Comment:
It would be nice if the tests added these if they are necessary.

Reply:
I think it's ok to require a schema be set up but data should be added by the tests